card/spec/lib/card/format/render_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe Card::Format::Render do
+  describe "view cache" do
+    before { Cardio.config.view_cache = true }
+
+    let(:cache_key) do
+      "a-Card::Format::HtmlFormat-normal-home_view:content;"\
+      "nest_name:A;nest_syntax:A|content;view:contentcontent:show"
+    end
+
+    subject { Card::Cache[Card::View] }
+
+    it "can be changed with nest option" do
+      is_expected.to receive(:fetch).with cache_key
+      render_content "{{A|content}}"
+      is_expected.not_to receive(:fetch)
+      render_content "{{A|cache:never}}"
```

Comment:
So my concerns with this approach:
1. I could be wrong, but I don't think there is anything conceptually special about the +companies card; I think it's a general bug, and we'd just be picking which appearances of that bug annoy us most, no?
2. In principle, there are some kinds of caching that I think wagneers may want to think about.  Eg, they might want to be able to do `cache: daily` on a complicated homepage or report.  But right now the caching concept is pretty heavy.
3.  I think it would need to be clearer what this means.  When you look at it, your first guess is that it means "never cache the homeview of this slot", but as implemented, it means "don't cache any view ever rendered within this slot".  So if it starts as content, then goes to history or edit, etc, none of those get cached.

It's a clever fix to an ugly bug, but now that I know the site is performing ok without view caching, I think we should probably keep it turned off until we get view caching working more reliably, at which point this feature won't be needed.